server/controllers/match/placementRequests/occupancyViewController.test.ts

```diff
@@ -47,7 +48,7 @@ describe('OccupancyViewController', () => {
   })
 
   beforeEach(() => {
-    jest.resetAllMocks()
+    jest.clearAllMocks()
```

Comment:
I'm wondering why the change to just clear the mocks rather than reset? Doesn't that leave the mock implementation intact?
It's not a biggie as all the mock implementations get set up again anyway.

Reply:
Using `resetAllMocks` results in the `describe` block on line 301 to fail if run consecutively -- not entirely sure why, I suspect the way we're mocking requests and such may play a part!